README.md

```diff
@@ -8,6 +8,16 @@
 [![Scorecard supply-chain security](https://github.com/OpenZeppelin/defender-sdk/actions/workflows/scorecard.yml/badge.svg)](https://github.com/OpenZeppelin/defender-sdk/actions/workflows/scorecard.yml)
 [![CI](https://github.com/OpenZeppelin/defender-sdk/actions/workflows/ci.yml/badge.svg)](https://github.com/OpenZeppelin/defender-sdk/actions/workflows/ci.yml)
 
+## ⚠️ Important Notice
+
+> **This SDK will be deprecated on July 1, 2026 and will no longer receive updates after that date.**
+>
+> OpenZeppelin Defender is currently undergoing migration. If you are using OpenZeppelin Relayer, you can use the new SDK: [OpenZeppelin Relayer SDK](https://github.com/OpenZeppelin/openzeppelin-relayer-sdk).
```

Comment:
You can also call out 

_While defender-sdk has [defender monitor client](https://github.com/OpenZeppelin/defender-sdk/tree/main/packages/monitor), OpenZeppelin Monitor ( Open Source version ) doesn't ship with any client as it's standalone binary/CLI tool._